core-bundle/src/Resources/contao/library/Contao/FeedItem.php

```diff
@@ -113,12 +118,29 @@ public function addEnclosure($strFile, $strUrl=null, $strMedia='enclosure')
 			$strUrl = Environment::get('base');
 		}
 
+		$fileUrl = $strUrl . System::urlEncode($strFile);
 		$objFile = new File($strFile);
 
+		$imageSize = StringUtil::deserialize($imageSize, true);
```

Comment:
This line is overwriting the method argument, which is not a good thing to do. Maybe you can use a different name for the unserialized content?

Reply:
aa374353b51684cecff5afe6fff22836b5e07b5f did not fix this issue, I'm afraid, because now both the argument and the variable are called `$size`. One should be called `$imageSize` and the other one `$size`.